ruby/vampires.rb

```diff
@@ -0,0 +1,102 @@
+#Coding program to gather informmation from user
+puts "what is the total number of applicants today"
+TOTALAPPLICANTS = gets.chomp.to_i

```

Comment:
lets name our variables with the proper Ruby convention of snake_case.  All lower case.  You see how this variable is a different color than the others?  This is because Ruby recognizes this as a different type of object that you will learn about later.

use `total_applicants`